src/agent/Server.cpp

```diff
@@ -62,6 +63,7 @@ void Server::SendData()
 
 void Server::CollectInfoAboutDiscs()
 {
-    m_health.SetStatus( GeneralHealth::GOOD);
+    SmartReader reader;
+    m_health = reader.ReadDisksStatus();
```

Comment:
Not an issue yet, but Server should ask for status via injected interface so we can have loose coupling here.

Reply:
This will be changed in future task.